performance_test/CMakeLists.txt

```diff
@@ -38,6 +38,13 @@ endif()
 # Use ODB support to create SQL database
 option(PERFORMANCE_TEST_ODB_FOR_SQL_ENABLED "Enable ODB for SQL" OFF)
 
+# Build code using ROS Eloquent packages
+option(ROS_ELOQUENT_ENABLED "Use ROS Eloquent" OFF)
```

Comment:
Is there a way for checking for ROS 2 Eloquent automatically? For example check the ROS_DISTRO environment variable or some constant in `rclcpp.hpp` (I don't know if the latter works, though).

Reply:
We can use the ros_distro variable or we can check the major/minor version of rclcpp package for example. What do you think? @daggarwa @esteve . I commited a change using the ros_distro now.

Reply:
@BMarchi did you do the commit today? The latest commit on this PR I can see is on Oct 17,2019

Reply:
> We can use the ros_distro variable or we can check the major/minor version of rclcpp package for example. What do you think? @daggarwa @esteve . I commited a change using the ros_distro now.

@BMarchi How long will it take to implement the check for major/minor version of rclpcpp ? If it takes a lot of time. We can push the ros_distro change for now. Let us know.

Reply:
Lets keep this discussion in the comment below